core/src/test/scala/org/apache/spark/scheduler/TaskSetManagerSuite.scala

```diff
@@ -428,7 +428,7 @@ class TaskSetManagerSuite extends SparkFunSuite with LocalSparkContext with Logg
     val clock = new ManualClock
     // We don't directly use the application blacklist, but its presence triggers blacklisting
     // within the taskset.
-    val blacklistTrackerOpt = Some(new BlacklistTracker(conf, clock))
+    val blacklistTrackerOpt = Some(new BlacklistTracker(null, conf, clock))
```

Comment:
Same comment regarding the null listener bus. I'd be more confident if you provided something here (even if it's a mocked bus you never actually check).